From f6b03161361d8d3e4b999faa37096f01776a9759 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Thu, 3 Nov 2011 13:58:16 +0800
Subject: [PATCH 0538/2463] ENGR00161312 - FEC: fix build warning

fix build warning:
> drivers/net/fec.c:435: warning: unused variable 'estatus'

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 drivers/net/fec.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/net/fec.c b/drivers/net/fec.c
index 9730f6e..7108240 100644
--- a/drivers/net/fec.c
+++ b/drivers/net/fec.c
@@ -381,7 +381,6 @@ fec_enet_tx(struct net_device *ndev)
 	struct  fec_ptp_private *fpp;
 	struct bufdesc *bdp;
 	unsigned short status;
-	unsigned long  estatus;
 	struct	sk_buff	*skb;
 
 	fep = netdev_priv(ndev);
@@ -428,8 +427,7 @@ fec_enet_tx(struct net_device *ndev)
 
 #if defined(CONFIG_ENHANCED_BD)
 		if (fep->ptimer_present) {
-			estatus = bdp->cbd_esc;
-			if (estatus & BD_ENET_TX_TS)
+			if (bdp->cbd_esc & BD_ENET_TX_TS)
 				fec_ptp_store_txstamp(fpp, skb, bdp);
 		}
 #elif defined(CONFIG_IN_BAND)
-- 
1.7.7.4

